src/graphql/resolvers/block-resolver.ts

```diff
@@ -11,7 +11,10 @@ class BlockQueryArgs extends PaginationArguments {
   headerId?: string;
 
   @Field(() => Int, { nullable: true })
-  height?: number;
```

Comment:
In order to allow finding blocks by a specific height, I think we need to keep height here alongside with min and max heights.

Reply:
I had something different on my mind: To find a block of a specific height, one must specify both min and max height equal to the desired height.
I think that makes it more complicated. I'll ever the code.
Let me know if you agree with my previous thought.

Reply:
You have a good point, although I think having a `height` param is something that makes queries a bit more straight forward, if you want to fetch a single block.

Reply:
I agree.
If everything else looks fine, we can do the merge )